### DISCUSSION TITLE
"Quiet" mode

### DISCUSSION THREAD

**Discussion by rozalily** [biomejs/biome#2322] [Ideas]
When there are no diagnostics, `biome check`/`lint`/`format` still print messages to the effect of `Checked 123 files in 456ms. No fixes needed.`

There doesn't appear to be any way to hide this. Most tools like this don't print anything by default when there is no meaningful output to show. We could certainly debate whether Biome should follow that pattern, but I think it should at least allow us to disable this message somehow (such as with a `--quiet` or `--diagnostics-only` option).

Today, this can be worked around by suppressing output from stdout, but this is different from most common tools (`tsc` is quiet by default, `prettier` has `--log-level silent`).

**Comment by ematipico** [comment]
Not printing anything because nothing was processed is **bad DX**. You always need to provide meaningful feedback for every user's action, otherwise they might think that something is broken.

I like both suggestions.
